### PR TITLE
Add basic sliding functionality

### PR DESCRIPTION
**Pull request by amanketebo**

### CODE REVIEW

**Comment by amanketebo** [comment]
Yo @Krevalz, what's your Xcode setting for:
trimming

In case it's not both checked, can you check those?  It'll make it so SwiftLint has less warnings.

File: Drive/Features/TrackDrive/DriveCoordinator.swift [hunk 1]
```diff
@@ -57,14 +58,9 @@ class DriveCoordinator: UIViewController, DriveDelegate, LocationProviderDelegat
...
         let driveVC = DriveVC.fromStoryboard()
-
-        view.addSubview(driveVC.view)
-        addChild(driveVC)
-        driveVC.view.translatesAutoresizingMaskIntoConstraints = false
-        driveVC.view.leftAnchor.constraint(equalTo: view.leftAnchor).isActive = true
-        driveVC.view.rightAnchor.constraint(equalTo: view.rightAnchor).isActive = true
-        driveVC.view.topAnchor.constraint(equalTo: view.bottomAnchor, constant: -150).isActive = true
-        driveVC.view.heightAnchor.constraint(equalToConstant: view.bounds.height * 0.66).isActive = true
+        slidingCardManager = SlidingCardManager(slidingViewController: driveVC,
```
Comment (Krevalz) [inline Drive/Features/TrackDrive/DriveCoordinator.swift:61]: I like this

File: Drive/Features/TrackDrive/SlidingCardManager.swift [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            slidingViewController.view.heightAnchor.constraint(equalToConstant: slidingViewController.fullHeight)
+        ])
+
+        // Set up sliding
+        let gestureRecognizer = UIPanGestureRecognizer(target: self,
+                                                       action: #selector(handlePanGesture(_:)))
+        self.slidingViewController.view.addGestureRecognizer(gestureRecognizer)
+    }
+
+    @objc private func handlePanGesture(_ gestureRecognizer: UIPanGestureRecognizer) {
```
Comment (Krevalz) [inline Drive/Features/TrackDrive/SlidingCardManager.swift:49]: This function get's called as you're sliding?

File: Drive/Location/DebugCoordinatesManager.swift [hunk 3]
```diff
@@ -62,10 +62,10 @@ class DebugCoordinatesManager {
         }
 
         let singleDrive = try? DriveService.jsonDecoder.decode(GetCoordinatesResponse.self, from: debugCoordinatesData)
-        
-        let clLocations = singleDrive?.results.map({ CLLocation(latitude: CLLocationDegrees($0.latitude), longitude: CLLocationDegrees($0.longitude)) })
 
-        return clLocations
+        return singleDrive?.results.map {
```
Comment (Krevalz) [inline Drive/Location/DebugCoordinatesManager.swift:66]: Good clean up

**Review by Krevalz — APPROVED** [review]

File: Drive/Features/TrackDrive/SlidingCardManager.swift (hunk 2 above)
Comment (amanketebo) [reply Drive/Features/TrackDrive/SlidingCardManager.swift:49]: Yup 👍🏽

**Pull request merged**